package.json

```diff
@@ -23,25 +23,41 @@
     "central",
     "services"
   ],
-  "author": "Dwolla",
+  "author": "Modusbox",
```

Comment:
Then please align to the following example: https://github.com/mojaloop/central-services-error-handling/blob/master/package.json#L27